File: HashTablePython/src/NodoHash.py
"""
Autores:
    -   Ruben 
    -   Adaya
    -   Elmer
"""

class NodoHash:
    def __init__(self, dato): 
        self.dato = dato
        self.siguiente = None
    def getDato(self):
        return self.dato
    def setDato(self,nuevoDato):
        self.dato = nuevoDato
    def getSiguiente(self):
        return self.siguiente
    def setSiguiente(self,nuevoSig):
        self.siguiente = nuevoSig

File: HashTableJava/src/NodoHash.java
package src;
public class NodoHash<T> {
    private T dato;
    private NodoHash<T> siguiente;

    public NodoHash(T dato) {
        this.dato = dato;
        this.siguiente = null;
    }

    public T getDato() {
        return dato;
    }

    public void setDato(T dato) {
        this.dato = dato;
    }

    public NodoHash<T> getSiguiente() {
        return this.siguiente;
    }

    public void setSiguiente(NodoHash<T> siguiente) {
        this.siguiente = siguiente;
    }
}
